public/src/components/channelManagement/supportLandingPage/variantLandingPagePreview.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import { Button, Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import Drawer from '@mui/material/Drawer';
+import Typography from '@mui/material/Typography';
+import { buildStorybookUrl } from '../helpers/dcrStorybook';
+import {
+  SupportLandingPageCopy,
+  SupportLandingPageVariant,
+} from '../../../models/supportLandingPage';
+
+// Mock prices data
+interface ProductPriceData {
+  Monthly: {
+    price: string;
+  };
+  Annual: {
+    price: string;
+  };
+}
+interface CountryGroupPriceData {
+  GuardianWeekly: ProductPriceData;
+  Digisub: ProductPriceData;
+}
+type Prices = {
+  [index: string]: CountryGroupPriceData;
+};
+
+interface SupportLandingPageProps {
+  tracking: {
+    ophanPageId: string;
+    platformId: string;
+    referrerUrl: string;
+    clientName: string;
+    abTestName: string;
+    abTestVariant: string;
+    campaignCode: string;
+    campaignId: string;
+    componentType: string;
+    products: string[];
+  };
+  isSupporter: boolean;
+  countryCode?: string;
+  prices?: Prices;
+  copy: SupportLandingPageCopy;
+}
+
+const anchor = 'bottom';
+
+const buildProps = (variant: SupportLandingPageVariant): SupportLandingPageProps => {
+  return {
+    tracking: {
+      ophanPageId: 'ophanPageId',
+      platformId: 'GUARDIAN_WEB',
+      clientName: 'frontend',
+      referrerUrl: 'https://www.theguardian.com/',
+      abTestName: 'abTestName',
+      abTestVariant: variant.name,
+      campaignCode: 'campaignCode',
+      campaignId: '',
+      componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+      products: [],
+    },
+    isSupporter: false,
+    copy: {
+      heading: 'Support the Guardian',
+      subheading: 'Available for everyone, funded by readers',
+    },
+    countryCode: 'GB',
+  };
+};
+
+const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
+  drawer: {
+    height: '400px',
+    bottom: 0,
+    top: 'auto',
+    width: '100%',
+  },
+  icon: {
+    width: '40px',
+    height: '40px',
+    background: palette.grey[900],
+    borderRadius: '50%',
+    color: 'white',
+    cursor: 'pointer',
+  },
+  hint: {
+    textAlign: 'center',
+    fontStyle: 'italic',
+    fontSize: '20px',
+  },
+  controlsContainer: {
+    position: 'fixed',
+    backgroundColor: palette.grey[100],
+    borderRadius: '4px',
+    top: spacing(3),
+    left: spacing(3),
+    padding: spacing(3),
+  },
+  iframe: {
+    width: '100vw',
+    height: '100vh',
+  },
+}));
+
+interface VariantLandingPagePreviewProps {
+  variant: SupportLandingPageVariant;
+  controls?: React.ReactElement;
+}
+
+const VariantLandingPagePreview: React.FC<VariantLandingPagePreviewProps> = ({
```

Comment:
This isn't going to work, I suggest removing any preview functionality for now

Reply:
That's a good point. I have some time so can take a look at implementing this...

Reply:
Bear in mind there are 2 types of preview in the RRCP:
1. "live" preview, which iframes the storybook from DCR
2. "web" preview, which links out to dotcom and forces the test variant with the querystring

I think to begin with at least just adding the web preview would be fine (though it doesn't have to be in this PR!)
I'm wondering how valuable iframing a storybook would be here - as this tool configures an entire webpage, not just a component like the epic.